### PR TITLE
Add raw information to ConversionMetrics

### PR DESCRIPTION
**Pull request by tarrow**
Adds raw creation, first and last edit times to ConversionMetrics.

Also uses non-absolute values for engagement and abandonment time
this accounts for edge cases where we import wikis such that they have edits
before they are created on this WBStack instance

### CODE REVIEW

File: tests/Routes/Wiki/ConversionMetricTest.php [hunk 1]
```diff
@@ -21,6 +22,8 @@ public function setUp(): void {
         Wiki::query()->delete();
         WikiSiteStats::query()->delete();
         WikiSetting::query()->delete();
+        Carbon::setTestNow(Carbon::parse('first day of October 2023'));
```
Comment (m90) [inline tests/Routes/Wiki/ConversionMetricTest.php:25]: I've never done this in PHP before, but do we need to "unfreeze" the time again in `tearDown` so that following tests don't get tripped up?

Comment (tarrow) [reply tests/Routes/Wiki/ConversionMetricTest.php:25]: right now all tests pass, but I think that is a wise suggestion

**Review by m90 — APPROVED** [review]
